### ISSUE TITLE
running problem

### ISSUE THREAD

**Issue by david-lev** [ipython/ipython#12780]
Hello, I am using Linux Mint 20, I installed ipython using the 'pip3 install ipython' command and I do manage to run the program but only up to a certain point. I also do not get suggestions with the tab key. I do not know what more details need to be presented here but I would love to send what is needed.
https://del.dog/covivippim.txt

```
$ python3 -c "import IPython; print(IPython.sys_info())" 
{'commit_hash': '223e783c4',
 'commit_source': 'installation',
 'default_encoding': 'utf-8',
 'ipython_path': '/home/david/.local/lib/python3.8/site-packages/IPython',
 'ipython_version': '7.19.0',
 'os_name': 'posix',
 'platform': 'Linux-5.4.0-62-generic-x86_64-with-glibc2.29',
 'sys_executable': '/usr/bin/python3',
 'sys_platform': 'linux',
 'sys_version': '3.8.5 (default, Jul 28 2020, 12:59:40) \n[GCC 9.3.0]'}

```
@meeseeksdev tag help wanted

**Comment by alexander-newton** [comment]
This is the issue I've been facing - it's because the file /IPython/terminal/ptutils.py doesn't import sys at the top. Easy fix devs :)

**Comment by Jongy** [comment]
See https://github.com/ipython/ipython/issues/12745 (and many other duplicate issues for this bug.. lol).

TL;DR `pip install -U 'jedi==0.17.2'` should do the trick.

**Comment by david-lev** [comment]
> See https://github.com/ipython/ipython/issues/12745 (and many other duplicate issues for this bug.. lol).
> 
> TL;DR `pip install -U 'jedi==0.17.2'` should do the trick.

Thanks!!
